src/TLuaInterpreter.cpp

```diff
@@ -1251,52 +1251,47 @@ int TLuaInterpreter::getMapEvents(lua_State * L){
     return 0;
 }
 
-int TLuaInterpreter::centerview( lua_State * L )
+int TLuaInterpreter::centerview(lua_State* L)
 {
-    Host * pHost = TLuaInterpreter::luaInterpreterMap[L];
-    if( ! pHost ) {
-        lua_pushnil( L );
+    Host* pHost = TLuaInterpreter::luaInterpreterMap[L];
+    if (!pHost) {
+        lua_pushnil(L);
         lua_pushstring(L, "centerview: NULL Host pointer - something is wrong!");
         return 2;
-    }
-    else if( ! pHost->mpMap || ! pHost->mpMap->mpRoomDB ) {
-        lua_pushnil( L );
-        lua_pushstring(L, "centerview: no map present or loaded!");
+    } else if (!pHost->mpMap || !pHost->mpMap->mpRoomDB || !pHost->mpMap->mpMapper) {
+        lua_pushnil(L);
+        lua_pushstring(L, "centerview: you haven't opened a map yet");
```

Comment:
No full-stop:

```lua
> table.insert()
stdin:1: bad argument #1 to 'insert' (table expected, got no value)
stack traceback:
	[C]: in function 'insert'
	stdin:1: in main chunk
	[C]: ?
> 
```

> I looked at those best practices, but perhaps this is a little too much:

Yes because it is too wordy - you want to get your point across as consicely as possible. This isn't a novel so people will be happier spending less time reading something if they can :)